src/catalog/abstract_catalog.cpp

```diff
@@ -71,21 +71,27 @@ AbstractCatalog::AbstractCatalog(const std::string &catalog_table_ddl,
   auto catalog_database_name = create_plan->GetDatabaseName();
   PELOTON_ASSERT(catalog_schema_name == std::string(CATALOG_SCHEMA_NAME));
   // create catalog table
-  Catalog::GetInstance()->CreateTable(
-      catalog_database_name, catalog_schema_name, catalog_table_name,
-      std::unique_ptr<catalog::Schema>(catalog_table_schema), txn, true);
+  Catalog::GetInstance()->CreateTable(txn,
```

Comment:
Unless this function also creates the `DataTable` object (which it shouldn't), maybe we should rename this to `CreateTableEntry`.

Reply:
Ironically, I think it does create the DataTable object (Not that it should)